lib/__tests__/timing.test.mjs

```diff
@@ -0,0 +1,51 @@
+import { jest } from '@jest/globals';
+import process from 'node:process';
+import timing from '../timing.mjs';
+
+describe('timing', () => {
+	describe('getListSize()', () => {
+		const OLD_ENV = process.env;
+
+		beforeEach(() => {
+			jest.resetModules();
+			process.env = { ...OLD_ENV };
+		});
+
+		afterAll(() => {
+			process.env = OLD_ENV;
+		});
+
+		it('should return minimum list size when TIMING is not set or has a small value', () => {
+			delete process.env.TIMING;
+			expect(timing.getListSize()).toBe(10);
+
+			process.env.TIMING = 'true';
+			expect(timing.getListSize()).toBe(10);
```

Comment:
This one i covered by `'foo'`.
i.e. it can be removed

What about replacing it with `''`?
i.e. test another falsy value

Reply:
`foo` is a meaningless value. `true` indicates a boolean was specified, which has a different meaning.

Reply:
i.e. a test for accidental coercions that may detect regressions